whisper/whisper-torchserve/config.yaml

```diff
@@ -0,0 +1,21 @@
+environment_variables: {}
+external_package_dirs: []
+model_metadata: {}
+model_name: Whisper Torchserve
+python_version: py310
+requirements:
+  - torch==2.1.0
+  - torchserve==0.9.0
+  - ffmpeg-python==0.2.0
+  - transformers==4.37.2
+  - nvgpu==0.10.0
+  - httpx==0.27.0
+resources:
+  accelerator: T4
+  use_gpu: true
+secrets: {}
+system_packages:
+  - ffmpeg
+  - openjdk-11-jdk
+runtime:
+  predict_concurrency: 128
```

Comment:
@htrivedi99 you feel good about this number? higher than I would expect; e.g. if the batch size is 4 in torchserver, then i think this means we'll have 31 batches (128/4 - 1) waiting. But i may be thinking about it wrong.

Reply:
I think it's probably okay to leave it be quite large (since truss is just proxying). the desired number of pending batches can be controlled in the deployment autoscaling settings using the target concurrency